### PR TITLE
CB-15716: Add additional check on backup/restore validation

### PR DESCRIPTION
**Pull request by kkalvagadda1**

### CODE REVIEW

**Review by yg-gd — APPROVED** [review]

**Review by sxxgrc — APPROVED** [review]
Seems simple enough, I was unaware this was a possible scenario.

**Review by doktoric — APPROVED** [review]

File: datalake/src/main/java/com/sequenceiq/datalake/flow/SdxReactorFlowManager.java [hunk 1]
```diff
@@ -163,7 +163,10 @@ private boolean shouldSdxBackupBePerformed(SdxCluster cluster) {
         } else if (isVersionOlderThan(cluster, "7.2.1")) {
             retVal = false;
             reason = "Unsupported runtime: " + cluster.getRuntime();
-        } else if (cluster.getCloudStorageFileSystemType().isGcs()) {
+        } else if (cluster.getCloudStorageFileSystemType() == null) {
```
Comment (doktoric) [inline datalake/src/main/java/com/sequenceiq/datalake/flow/SdxReactorFlowManager.java:166]: maybe a unit test would be great

Comment (sodre90) [reply datalake/src/main/java/com/sequenceiq/datalake/flow/SdxReactorFlowManager.java:166]: +1

**Comment by sodre90** [comment]
looks good to me, please create a simple unit as @doktoric suggested.

File: datalake/src/main/java/com/sequenceiq/datalake/flow/SdxReactorFlowManager.java (hunk 1 above)
Comment (kkalvagadda1) [reply datalake/src/main/java/com/sequenceiq/datalake/flow/SdxReactorFlowManager.java:166]: added

**Comment by kkalvagadda1** [comment]
> looks good to me, please create a simple unit as @doktoric suggested.

Added unit to test for the same.

**Pull request merged**